aws-ts-url-shortener-cache-http/index.ts

```diff
@@ -2,12 +2,9 @@
 
 import * as pulumi from "@pulumi/pulumi";
 import * as aws from "@pulumi/aws";
-import * as awsx from "@pulumi/awsx";
 import * as cloud from "@pulumi/cloud-aws";
```

Comment:
We can drop cloud here now?

Reply:
not just yet.  let me ping you about this.